src/encoded/regulome_search.py

```diff
@@ -533,7 +534,13 @@ def region_search(context, request):
         allowed_status = atlas.allowed_statuses()
         facets = _REGULOME_FACETS
 
-        query = get_filtered_query('Dataset', [], set(), principals, atlas.set_type())
+        query = get_filtered_query(
+            'Dataset',
+            [],
+            sorted(list_result_fields(request, ['Experiment', 'Annotation'])),
+            principals,
+            atlas.set_type()
+        )
```

Comment:
Not sure how this work. The third argument and the `list_result_fields` will make elastic return less or trim returns from elastic?

Reply:
list_result_fields will look at the 'columns' for the specified doc_types in the schema.   Empty list (which is what code use to day) defaults to "*".